.zuul.yaml

```diff
@@ -0,0 +1,10 @@
+- project:
+    check:
+      jobs:
+        - thoth-coala
+    gate:
+      jobs:
+        - thoth-coala
+    kebechet-auto-gate:
+      jobs:
+        - thoth-coala
```

Comment:
What about releases? I will add sesheta to https://pypi.org/project/thoth-python/

Reply:
it was not on the trello card :P

Reply:
#needmorepractice, sesheta added to the PyPI project